src/Codeception/Extension/MultiDb.php

```diff
@@ -1200,7 +1200,8 @@ function ($value) use (&$param_list, $driver) {
                     },
                     $criteriaParams
                 )
-            )
+            ), 
+            $limit
```

Comment:
I am not sure if this would work; have you tried running it successfully?
I can only see the `$limit` number; doesn't seem to add the `LIMIT` keyword.

i.e the result for
```
$I->getFromDb('Db1.Table1', ['Field1'=>123], null, 1)
```
will look like
```
SELECT * FROM Db1.Table1 1
```
Rather than
```
SELECT * FROM Db1.Table1 LIMIT 1
```